src/components/ebay-pagination/README.md

```diff
@@ -0,0 +1,53 @@
+# ebay-pagination
+
+The `<ebay-pagination>` is a tag used to create a pagination navigation.
+
+## ebay-pagination Tag
+
+### ebay-pagination Usage
+
+```marko
+<ebay-pagination aria-label-prev="Previous page" aria-label-next="Next page" curr-text="Results Pagination - Page 2">
+    <ebay-pagination-item href="#" previous disabled/>
```

Comment:
I'm not sure if there is any benefit in letting the developer declaratively add the prev and next items? Can't we abstract away that detail for them?

Reply:
@ianmcburnie FYI there was some discussion of this here: https://github.com/eBay/ebayui-core/pull/100#discussion_r182160536

Reply:
Thanks. Ok, let's go with the tag for now if we need to add attributes to it, but I'm not a fan of them being booleans, because it would never make sense to have them both set to true. Maybe `type="item(default)|prev|next"`?

Reply:
Okay I will change it to type instead of a boolean.